packages/server/api/src/app/flows/flow/flow-v2.controller.ts

```diff
@@ -44,6 +45,36 @@ export const flowV2Controller: FastifyPluginAsyncTypebox = async (app) => {
       throw error;
     }
   });
+
+  app.put('/:id/steps/:stepName', UpdateStepRequestOptions, async (request) => {
```

Comment:
as with the other endpoint, its basically what happens in the original endpoint of post flow but this way the ai doesnt get immediate access to all the post logic